utils/clone-rocm-air-platforms.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+##===- utils/clone-rocm-air-platforms.sh ---------------------*- Script -*-===##
+#
+# Copyright (C) 2022, Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: MIT
```

Comment:
```
# Copyright (C) 2022, Advanced Micro Devices, Inc.
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Reply:
Thanks, addressed in [f18a8d6](https://github.com/Xilinx/mlir-aie/pull/570/commits/f18a8d66b255ed857b56380a62fe27e5ba046e25).